plane/source/docs/quadplane-flying.rst

```diff
@@ -7,47 +7,58 @@ While flying a QuadPlane can actually be easier than flying a
 conventional fixed wing aircraft there are some things you need to
 understand. Please read the following sections carefully.
 
-Transition
-==========
+Transitions
+===========
 
 QuadPlane transition is where the aircraft is changing between flying
 primarily as a VTOL (copter-like) aircraft and flying as a
 conventional fixed wing aircraft. Transition happens in both
 directions, and can either be commanded by the pilot or happen
-automatically based on airspeed and flight mode.
+automatically based on airspeed/attitude/altitude(:ref:`assisted_fixed_wing_flight` or flight mode changes during a mission.
 
 The primary way to initiate a transition is to change flight mode,
 either using the flight mode channel on your transmitter or using a
 ground station to command a mode change.
 
--  If you transition to :ref:`MANUAL <manual-mode>` then the VTOL motors
-   will immediately stop. In the case of a tilt-rotor, the motors will immediately rotate to foward flight orientation.
+During a transition, depending on type of QuadPlane and various :ref:`Q_OPTIONS<Q_OPTIONS>` settings, pilot control of the vehicle's attitude, climb rate, etc. may be modified or restricted to assure successful transition as detailed below.
+
+Transition to Fixed Wing Mode from VTOL
+---------------------------------------
```

Comment:
I wonder if its worth a warning that, as ever, its all different for tailsitters.

Reply:
added